docs/UserGuide.md

```diff
@@ -78,7 +82,7 @@ An inventory item can have any number of tags (including 0). A tag should only c
 
 Examples:
 * `addi n/Keychain q/20 d/Silicone keychain with a metal buckle sp/3.50 cp/1`
-* `addi n/Chair q/10 d/This is a wooden dining chair t/Furniture sp/50 cp/20`
+* `addi n/Chair q/10 d/This is a wooden dining chair t/Furniture t/Mahogany sp/50 cp/20`
```

Comment:
I remember one of the bugs was that the format in the UG and the application was inconsistent for the `addi` command, for the item name prefix. It should be `i/` instead of `n/` here. Do remember to change this for all relevant sections in the UG